addons/web/static/src/views/helpers/sample_server.js

```diff
@@ -309,7 +309,7 @@ export class SampleServer {
      */
     _getRandomDate(format) {
         const delta = Math.floor((Math.random() - Math.random()) * SampleServer.DATE_DELTA);
-        return luxon.DateTime.local().plus({ hours: delta }).toFormat(format);
+        return luxon.DateTime.local().plus({ hours: delta }).toFormat(format, { numberingSystem:'latn' });
```

Comment:
I'd rather have you use the `serializeDate` and `serializeDateTime` from `@web/core/l10n/dates` to keep a consistent center point to format server dates. To do so you can just return the DateTime object in this function and adapt the `switch ... case` statements higher up to use the correct helpers

Reply:
And also adapt the inline doc (returns ~~moment~~ -> DateTime)

Reply:
@Arcasias  Indeed better. Thanks! :+1: